src/app/tabs/compile-tab.js

```diff
@@ -1,6 +1,6 @@
-/* global */
-var $ = require('jquery')
 var yo = require('yo-yo')
+var remixLib = require('remix-lib')
```

Comment:
why do you had these dependencies ?

Reply:
i will check - maybe refactoring made it unnecessary. i'll remove it.